### ISSUE TITLE
Validation Accuracy

### ISSUE THREAD

**Issue by anerisheth19** [DhruvJawalkar/PyTorch-OpenPose-Realtime-Multi-Person-2D-Pose-Estimation-using-Part-Affinity-Fields#2]
Can you please report your validation accuracy for network resolution of 368x368?

**Comment by rabysam28** [comment]
Impressive work @DhruvJawalkar!
I was also wondering if the accuracy of this pytorch model would be the same as the one in caffe reported in Dec 2018? Thanks!
